File: AQ_LIB/resources/api/RExamples/CurveMarketDataDisplay/M_CurveMarketDataDisplayFromCurve.R
dyn.load("MLIB_CLIENT_API")
source("MLIB_CLIENT_API.R")
getLoadedDLLs()
setUpMLIB("","")
meUtilityVersion()


filename <-c("E:/SavedCurves/USD_OIS_CURVE.TXT")
oisCurve <- meLWOLoad(filename)

blockname <- c("SWAPS")
block <- meLWOCurveMarketDataDisplayFromCurve(oisCurve, blockname)

block

File: AQ_LIB/resources/api/PythonExamples/CurveMarketDataDisplay/M_CurveMarketDataDisplayFromCurve.py
#Import the MLIB CLIENT API
import MLIB_CLIENT_API as m

#Load Calendars and Config Files
m.setUpMLIB("","")

#Get the MLIB Version
m.meUtilityVersion()


# Display a block of data from a saved curve

curveFilename="E:\SavedCurves\USD_OIS_CURVE.TXT"
oisCurve = m.meLWOLoad(curveFilename)

blockname = "SWAPS"
block= m.meLWOCurveMarketDataDisplayFromCurve( oisCurve, blockname)

for row in block:
  for item in row:
    print item,
  print
  
